test/e2e/pom.xml

```diff
@@ -44,6 +44,8 @@
     </modules>
 
     <properties>
+        <sw.version>8.5.0-SNAPSHOT</sw.version>
```

Comment:
```suggestion
        <sw.version>${env.SKYWALKING_VERSION}</sw.version>
```

How about this? And update the E2E doc a little to match this?

Reply:
> ${env.SKYWALKING_VERSION}

With this, the local IDE won't work unless you manually replace this with the existing version, you cannot even set environment variable in the IDE (see my screenshot below).

<img width="1791" alt="image" src="https://user-images.githubusercontent.com/15965696/108598565-44832c00-73c9-11eb-9af5-bc67f4158621.png">

If the developers have to replace `${env.SKYWALKING_VERSION}` with an actual version, it's highly possible it would be checked into the codebase.

Reply:
Strange thing, I read this from https://books.sonatype.com/mvnref-book/reference/resource-filtering-sect-properties.html

Reply:
> Strange thing, I read this from https://books.sonatype.com/mvnref-book/reference/resource-filtering-sect-properties.html

It works from command line, but not in IDEA

Reply:
Doesn't this work? 

![image](https://user-images.githubusercontent.com/5441976/108599495-69c56980-73cc-11eb-8bfc-bc8ac4c1cf53.png)

Reply:
> Doesn't this work?
> 
> ![image](https://user-images.githubusercontent.com/5441976/108599495-69c56980-73cc-11eb-8bfc-bc8ac4c1cf53.png)

This is running configuration (take effect in Java application or JUnit test case), not in maven's lifecycle, and, the pom.xml takes effect at compile phase.

Reply:
I can adopt `${env.SKYWALKING_VERSION}` and update the doc, but we need to pay attention to PRs that when developers modify `${env.SKYWALKING_VERSION}` to something like `8.5.0-SNAPSHOT`, it's not checked into the codebase.

Reply:
> but we need to pay attention to PRs that when developers modify ${env.SKYWALKING_VERSION} to something like 8.5.0-SNAPSHOT

How about we add a step in the CI to make sure the version is `${env.SKYWALKING_VERSION}`(or EMPTY?), before we set it by ourselves.

Reply:
No need to do that, check my latest commit